ui.frontend/src/main/components/App/config.js

```diff
@@ -22,6 +22,7 @@ export default {
         minicart: '#minicart'
     },
     pagePaths: {
-        addressBook: '/content/venia/us/en/my-account/address-book.html'
+        addressBook: '/content/venia/us/en/my-account/address-book.html',
+        baseUrl: document.querySelector('.header__logo').parentElement.pathname
```

Comment:
I'm not sure if this is a reliable way to detect the home-page of the store.

Reply:
Actually the path should also not be here. Assume someone us using it with its own site and use a different site name or  change the `my-account` page name.

Reply:
Got it. What's the best way to detect the homepage of the store, and where should I set this?

Reply:
I don't know for JS, unfortunately. It is not that simple I think. In Java there are Util classes for to get site root path or URL, which will also consider URL rewrites etc.

Reply:
@herzog31  thoughts? initially i thought if we can safely assume the logo is a constant element that redirects to the storefront, i could use it. but it looks like that assumptions not safe anymore like Markus said

Reply:
Just an idea, we could do it this way:
* Get the base url like in the header component: https://github.com/adobe/aem-core-cif-components/blob/master/bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/header/HeaderImpl.java#L55-L66
* Add it to the `StoreViewExporter`: https://github.com/adobe/aem-core-cif-components/blob/master/bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/storeconfigexporter/StoreConfigExporter.java
* Expose it via the page: https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/page/v1/page/page.html#L24
* Consume via JavaScript.

@dplaton @mhaack @alkim91 wdyt?

Reply:
Yes makes sense to me. That way we keep it consistent via the `StoreConfigExporter` all config related attributes in one place.

Reply:
So we'd just configure all the paths in AEM? I'm ok with that, although it requires a different JIRA issue for refactoring the Address Book path as well.

Reply:
@dplaton We have those already at https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/page/v1/page/_cq_dialog/.content.xml#L76-L89. We just don't expose them _yet_ :)

Reply:
Oh, we have _some_ of those :)
Ok, let's expose them.

Reply:
Thanks for the suggestions! I'll re-implement today 👍